src/sentry/static/sentry/app/components/lightWeightNoProjectMessage.tsx

```diff
@@ -17,7 +17,11 @@ export default class LightWeightNoProjectMessage extends React.Component<Props>
   };
 
   render() {
-    return (
+    const {organization} = this.props;
+
+    return organization.projects ? (
+      <NoProjectMessage {...this.props} />
```

Comment:
@davidenwang This ternary seems like the conditions should be reversed.

Reply:
What i'm trying to accomplish with this ternary is if `organization.projects` is valid as in the organization object has projects then we should use the regular `<NoProjectMessage>` otherwise if `projects` does not exist in `organization` we will have to fetch projects separately to see if we can show the <NoProjectMessage>

Reply:
I understand it's a little confusing becuase it seems like im saying "if there are projects in organization show the user the no project message" But the no project message evaluates if there are projects within the organization.projects array.

Reply:
@davidenwang I see. The problem is that an empty array in JS is truthy, and that according to the TS type of `Organization` that `organization.projects` will never be a falsy value. Is the TS type incorrect?

Reply:
Ahhh yes this TS type might be outdated perhaps it would be useful to have a lightweight and a heavyweight/detailed organization as the intention is to make it so that teams and projects are decoupled from organization

Reply:
@davidenwang Yes, I think it makes sense to make two different types. We can use the union of the types when we don't know what we're getting.

Reply:
Sounds good should I make that in a separate PR? I don't want to make this one too big.

Reply:
@davidenwang Your call. I don't think this PR is too big with the change. But I don't think we should approve this PR until the typing of `Organization` matches the usage you've added.